english/bite-the-marshmallow/tasty-marshmallow.md

```diff
@@ -2,6 +2,12 @@
 
 You close your eyes.
 
+Suddenly there is a horrible pain in your mouth! Who would have
+thought marshmallows, so white, so innocent, could cause such excruciating pain! But come to think of it, you've never gone
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.